ssn/integrated/oldssn.ttl

```diff
@@ -530,24 +530,6 @@ ssn:Drift rdf:type owl:Class ;
           rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Sensor#MeasuringCapability" .
 
 
-###  http://www.w3.org/ns/ssn/FeatureOfInterest
```

Comment:
Maxime, can you give me a hand here? I am probably missing something but where does the lower case "feature of interest" label comes from in comparison to the upper case version we used before. Thanks.